buku

```diff
@@ -1148,14 +1160,13 @@ class BukuDb:
 
         # If reading from DB, show empty title and desc as empty lines. We have to convert because
         # even in case of add with a blank title or desc, '' is used as initializer to show '-'.
-        result = edit_rec(editor, rec[1], rec[2] if rec[2] != '' else None,
-                          rec[3], rec[4] if rec[4] != '' else None)
+        result = edit_rec(editor, rec.url, rec.title or None, rec.tags, rec.desc or None)
```

Comment:
Please confirm empty rec.title fails the check.

Reply:
https://docs.python.org/3/library/stdtypes.html#truth-value-testing